Source/UI/Home/HomeView.swift

```diff
@@ -0,0 +1,314 @@
+//
+//  HomeView.swift
+//  Planetary
+//
+//  Created by Martin Dutra on 27/11/22.
+//  Copyright © 2022 Verse Communications Inc. All rights reserved.
+//
+
+import Analytics
+import CrashReporting
+import Logger
+import SwiftUI
+
+struct HomeView: View {
+    
+    private var feedStrategyStore = FeedStrategyStore()
+
+    @EnvironmentObject
+    private var botRepository: BotRepository
+
+    @EnvironmentObject
+    private var helpDrawerState: HelpDrawerState
+
+    private let helpDrawer = HelpDrawer.home
+
+    @State
+    private var messages: [Message]?
+
+    @State
+    private var isLoadingFromScratch = false
+
+    @State
+    private var isLoadingMoreMessages = false
+
+    @State
+    private var offset = 0
+
+    @State
+    private var noMoreMessages = false
+
+    @State
+    private var numberOfNewItems = 0
+
+    @State
+    private var lastTimeNewFeedUpdatesWasChecked = Date()
+
+    @State
+    private var errorMessage: String?
+
+    private var shouldShowAlert: Binding<Bool> {
+        Binding {
+            errorMessage != nil
+        } set: { _ in
+            errorMessage = nil
+        }
+    }
+
+    private var shouldShowFloatingButton: Bool {
+        numberOfNewItems > 0
+    }
+
+    var body: some View {
+        ZStack(alignment: .top) {
+            Group {
+                if let messages = messages {
+                    ScrollView(.vertical, showsIndicators: false) {
+                        ZStack {
+                            LazyVStack(alignment: .center) {
```

Comment:
I think I saw this same code in `HashtagView`. Maybe we could refactor it into a `MessageList` component?

Reply:
Being done in a separate PR